target/linux/pistachio/marduk/target.mk

```diff
@@ -13,7 +13,7 @@ DEFAULT_PACKAGES+=kmod-leds-gpio kmod-ledtrig-heartbeat kmod-i2c-core i2c-tools
                   kmod-tpm-i2c-infineon kmod-random-tpm tpm-tools \
                   uhttpd uboot-envtools tcpdump board-test \
                   luci kmod-bluetooth kmod-mac802154 \
-                  curl libopenssl ca-certificates
+                  curl libopenssl ca-certificates kmod-w1
```

Comment:
This kmod isn't all that useful on its own so whats the point in having it in the default packages, best to just drop completely?